src/app/wizard/versions/versions.md

```diff
@@ -2,8 +2,33 @@
 
 ## Overview
 
-Obsidian Toaster is a cloud application platform that includes a collection of cloud services with language runtime integration running on OpenShift.
-This initial release delivers a project generator that creates a quickstart or custom Spring Boot / Vert.x starter project, optionally secured with Red hat SSO, and is ready to be built and deployed to OpenShift.
+Obsidian Toaster defines and implements an opinionated approach to creating and running polyglot applications optimized for a common set of services running on OpenShift.
+
+## What's New in Alpha 2
+
+__WildFly Swarm__
+Adds WildFly Swarm (version 2017.1.1) as a language runtime.
```

Comment:
The version used in the Starter project is 2017.2.0, maybe the quickstarts should follow the same version?